### PR TITLE
Updates to font awesome

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,13 +2,13 @@
   "dependencies": {
     "@cquiroz/aladin-lite": "0.7.6",
     "@fortawesome/fontawesome-pro": "^6.0.0",
```
Comment (toddburnside) [inline package.json:4]: Should this be 6.1.2, as well?

File: package.json [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
-    "@fortawesome/fontawesome-svg-core": "^1.3.0",
-    "@fortawesome/pro-duotone-svg-icons": "^6.1.1",
-    "@fortawesome/pro-light-svg-icons": "^6.1.1",
-    "@fortawesome/pro-regular-svg-icons": "^6.1.1",
-    "@fortawesome/pro-solid-svg-icons": "^6.1.1",
-    "@fortawesome/pro-thin-svg-icons": "^6.1.1",
-    "@fortawesome/react-fontawesome": "^0.1.18",
+    "@fortawesome/fontawesome-svg-core": "^6.1.0",
```
Comment (toddburnside) [inline package.json:5]: Should this be 6.1.2?

**Review by toddburnside — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (cquiroz) [reply package.json:4]: yes, I'll fix that

**Pull request merged**